src/analyses/libraryFunctions.ml

```diff
@@ -44,6 +44,14 @@ let c_descs_list: (string * LibraryDesc.t) list = LibraryDsl.[
     ("setbuf", unknown [drop "stream" [w]; drop "buf" [w]]);
     ("swprintf", unknown (drop "wcs" [w] :: drop "maxlen" [] :: drop "fmt" [r] :: VarArgs (drop' [])));
     ("assert", special [__ "exp" []] @@ fun exp -> Assert { exp; check = true; refine = get_bool "sem.assert.refine" }); (* only used if assert is used without include, e.g. in transformed files *)
+    ("difftime", unknown [drop "time1" []; drop "time2" []]);
+    ("system", unknown [drop "command" [r]]);
+    ("wcscat", unknown [drop "dest" [r; w]; drop "src" [r]]);
+    ("abs", unknown [drop "j" []]);
+    ("localtime_r", unknown [drop "timep" [r]; drop "result" [w]]);
+    ("strsep", unknown [drop "stringp" [r_deep; w]; drop "delim" [r]]);
```

Comment:
This one should probably go to glibc or linux, it is not standard mandated: https://man7.org/linux/man-pages/man3/strsep.3.html

Reply:
True, I think I just saw `string.h` and thought it would be standard. I moved it to glibc because it's listed here: https://www.gnu.org/software/gnulib/manual/html_node/Glibc-string_002eh.html.